composer_migration/src/composer_migration/lib/strategies/migrate_kubernetes_pod_operator_ast.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2023 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, TypeVar
+import ast
+import logging
+
+from composer_migration.lib.strategies.strategy import DAGChecker
+
+# https://peps.python.org/pep-0484/#annotating-instance-and-class-methods
+T = TypeVar("T", bound="CheckKubernetesPodOperator")
+A = TypeVar("A", bound="ast.AST")
+
+
+# just visit needed nodes per
+# https://greentreesnakes.readthedocs.io/en/latest/manipulating.html#working-on-the-tree
+# subclass visitor for what we need
+# also derived from IBIS https://github.com/ibis-project/ibis-bigquery/blob/main/ibis_bigquery/udf/core.py
+
+# KubernetesPodOperator objects will only ever be under an "Assign"
+# This traverses the tree and adds all "Assign" nodes to a list
+class AssignFinder(ast.NodeVisitor):
+    def __init__(self: ast.NodeVisitor) -> None:
+        self.possible_kpo_nodes = []
+
+    def visit_Assign(self: ast.NodeVisitor, node: ast.Assign) -> None:
```

Comment:
Why do some of the methods have a big letter in the second word? I think we should use normal snake case for all of them

Reply:
👍